utils/utils.go

```diff
@@ -48,6 +49,42 @@ func Min(a int, b int) int {
 	return b
 }
 
+func HostZoneUnbondingKeys(m map[string]*recordstypes.HostZoneUnbonding) []string {
+	keys := make([]string, 0, len(m))
+	for k := range m {
+		keys = append(keys, k)
+	}
+	sort.Strings(keys)
+	return keys
+}
+
+func StringToIntMapKeys(m map[string]int64) []string {
```

Comment:
I think you can collapse these functions into one using this approach
https://go.dev/play/p/g8BXwgQq14S

Reply:
I think this does work, but the linter doesn't like it